<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InscriptionCandidatClass</class>
 <widget class="QDialog" name="InscriptionCandidatClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InscriptionCandidat</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Candidat_Prenom">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero civic:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Province:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Candidat_Nom">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Candidat_Nas">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Candidat_CodePostal">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Candidat_Ville">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Candidat">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Candidat_Province">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Candidat_OK">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Candidat_NumeroCivic">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parti politique:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Candidat_NomRue">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom rue:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Candidat_PartiPolitique">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>261</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inscription dun candidat</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Candidat_OK</sender>
   <signal>clicked()</signal>
   <receiver>InscriptionCandidatClass</receiver>
   <slot>validerInscriptionCandidat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerInscriptionCandidat()</slot>
 </slots>
</ui>
